lib/convert-yaml.js

```diff
@@ -0,0 +1,11 @@
+const YAML = require('yaml')
```

Comment:
This file is what I saw and commented on where it was imported.  Sorry for the confusion.

Reply:
The last thing to follow up on is to remove this file I think, since it was inlined where it is used.